commands/hseval/hseval.js

```diff
@@ -0,0 +1,25 @@
+const spawn = require('child_process').spawn;
+
+function hseval(code, cb) {
+  let out = '';
+  const mueval = spawn('mueval', ['--inferred-type', '-e', code]);
+  mueval.stdout.on('data', (d) => {
+    out += d.toString();
+  });
+  mueval.stderr.on('data', (d) => {
+    out += d.toString();
+  });
+  mueval.on('exit', () => {
+    cb(out);
+  });
+}
+module.exports = {
+  usage: 'hseval <haskell expression> - runs <haskell expression> using mueval as the sandbox',
+  run: (bot, message, args) => {
+    if (!args.length) return true;
```

Comment:
Oh, true! That is correct. I was just being explict in my code because thats how i was thinking at the time